project/Dependencies.scala

```diff
@@ -197,6 +197,16 @@ object Dependencies {
     "com.azure.resourcemanager" % "azure-resourcemanager" % "2.18.0"
   )
 
+  val wsmDependencies: List[ModuleID] = List(
+    "bio.terra" % "workspace-manager-client" % "0.254.452-SNAPSHOT"
+      exclude("jakarta.xml.bind", "jakarta.xml.bind-api")
+      exclude("jakarta.activation", "jakarta.activation-api")
+      exclude("jakarta.activation", "jakarta.activation")
+      exclude("jakarta.annotation", "jakarta.annotation-api")
+      exclude("com.sun.activation", "jakarta.activation")
```

Comment:
I was absolutely convinced there had to be at least one dupe in here but no, `annotation` & `activation` and `jakarta` & `com.sun` are just conspiring to line up and give me a headache.

Reply:
Yeah, the near-repetition here is what made this take so long, I was like "I'm already excluding all these things, why isn't it working???"